ecommerce/enterprise/utils.py

```diff
@@ -529,42 +529,29 @@ def has_enterprise_offer(basket):
     return False
 
 
-def get_enterprise_catalog(site, enterprise_catalog, limit, page, endpoint_request_url=None):
+def get_enterprise_catalog(site, enterprise_catalog, page_size, page, endpoint_request_url=None):
     """
     Get the EnterpriseCatalog for a given catalog uuid.
 
     Args:
         site (Site): The site which is handling the current request
         enterprise_catalog (str): The uuid of the Enterprise Catalog
-        limit (int): The number of results to return per page.
+        page_size (int): The number of results to return per page.
         page (int): The page number to fetch.
         endpoint_request_url (str): This is used to replace the lms url with ecommerce url
 
     Returns:
         dict: The result set containing the content objects associated with the Enterprise Catalog.
         NoneType: Return None if no catalog with that uuid is found.
-
-    Response will look like
-
-        {
-            "uuid": "2c5b9176-42be-4a3a-ad73-f1c1a09d2734",
-            "title": "enterprise catalog title",
-            "enterprise_customer": "67460e74-02e3-11e8-b443-00163e990bdb",
-            "catalog_query_uuid": None,
-            "content_last_modified": None,
-            "catalog_modified": "2022-05-26T12:56:20.346837Z",
-            "query_title": None
-        }
-
     """
     resource = 'enterprise-catalogs'
     partner_code = site.siteconfiguration.partner.short_code
-    cache_key = u'{site_domain}_{partner_code}_{resource}_{catalog}_{limit}_{page}'.format(
+    cache_key = u'{site_domain}_{partner_code}_{resource}_{catalog}_{page_size}_{page}'.format(
```

Comment:
u  - prefix could be omitted since we are on py3.x or even better we could use f-string instead of formated one.